docs/netdata-agent/versions-and-platforms.md

```diff
@@ -1,116 +1,235 @@
 # Netdata Agent Versions & Platforms
 
-Netdata is evolving rapidly and new features are added at a constant pace. Therefore, we have a frequent release cadence to deliver all these features to use as soon as possible.
+Netdata is evolving rapidly and new features are added at a constant pace. Therefore, we have a frequent release cadence to deliver all these features to you as soon as possible.
 
-Netdata Agents are available in 2 versions:
+You can choose from 2 Netdata Agent versions:
 
 | Release Channel |               Release Frequency               |                 Support Policy & Features                 |             Support Duration             |                              Backwards Compatibility                              |
 |:---------------:|:---------------------------------------------:|:---------------------------------------------------------:|:----------------------------------------:|:---------------------------------------------------------------------------------:|
-|     Stable      | At most once per month, usually every 45 days | Receiving bug fixes and security updates between releases | Up to the 2nd stable release after them  |     Previous configuration semantics and data are supported by newer releases     |
-|     Nightly     |           Every night at 00:00 UTC            |               Latest pre-released features                | Up to the 2nd nightly release after them | Configuration and data of unreleased features may change between nightly releases |
+|   **Stable**    | At most once per month, usually every 45 days | Receiving bug fixes and security updates between releases | Up to the 2nd stable release after them  |     Previous configuration semantics and data are supported by newer releases     |
+|   **Nightly**   |           Every night at 00:00 UTC            |               Latest pre-released features                | Up to the 2nd nightly release after them |Configuration and data of unreleased features may change between nightly releases|
```

Comment:
This should be ‘most nights around 02:00 UTC’.

We only actually publish nightlies if there are new changes since the last nightly build, and while the builds start at midnight UTC, they actually become available to users around 02:00 UTC.